addon/components/nypr-ui/brand-header.js

```diff
@@ -1,19 +1,75 @@
 import Component from '@ember/component';
 import layout from '../../templates/components/nypr-ui/brand-header';
+import {getOwner} from '@ember/application';
+import {computed} from '@ember/object';
+
 export default Component.extend({
   layout,
   tagName: '',
   init() {
     let menuItems = [
-      {name: "WNYC", url: "https://www.wnyc.org"},
-      {name: "Gothamist", url: "https://www.gothamist.com"},
-      {name: "WNYC Studios", url: "https://www.wnycstudios.org"},
-      {name: "NJPR", url: "http://www.njpublicradio.org"},
-      {name: "WQXR", url: "https://www.wqxr.org"},
-      {name: "New Sounds", url: "https://www.newsounds.org"}
+      {
+        name: 'WNYC',
+        slug: 'wnyc',
+        url: `https://www.wnyc.org?utm_medium=partnersite&utm_campaign=brandheader&utm_source=${this.get(
+          'siteSlug',
+        )}`,
+        svg: 'wnyc-logo',
+      },
+      {
+        name: 'Gothamist',
+        slug: 'gothamist',
+        url: `https://www.gothamist.com?utm_medium=partnersite&utm_campaign=brandheader&utm_source=${this.get(
+          'siteSlug',
+        )}`,
+        svg: 'gothamist-logo',
+      },
+      {
+        name: 'WNYC Studios',
+        slug: 'wnycstudios',
+        url: `https://www.wnycstudios.org?utm_medium=partnersite&utm_campaign=brandheader&utm_source=${this.get(
+          'siteSlug',
+        )}`,
+        svg: 'wnyc-studios-logo',
+      },
+      {
+        name: 'NJPR',
+        slug: 'njpr',
+        // Use a static UTM code for njpr since siteSlug will return wnyc
```

Comment:
I believe the `utm_source` param needs to be the value of the site that's currently loaded, not the destination, so if you're on www.wqxr.org, all of these links should have their `utm_source` as `wqxr`.